resources/versioning.md

```diff
@@ -0,0 +1,90 @@
+# Versioning policy
+
+This page describes the versioning rules Meilisearch SDKs follow after the release of Meilisearch engine v1.0.0. It explains how and when the Integrations Team should increase the MAJOR, MINOR, and PATCH of the versions.
+
+## 🤖 Basic rules
+
+Meilisearch SDKs releases follow the [SemVer rules](https://semver.org/), including the following basic ones:
+
+> 🔥 Given a version number MAJOR.MINOR.PATCH, increment the:
+>
+> 1. MAJOR version when you make incompatible API changes
+> 2. MINOR version when you add functionality in a backwards compatible manner
+> 3. PATCH version when you make backwards compatible bug fixes
+
+
+**Changes that MAY lead the Meilisearch users (developers) to change their code are considered API incompatibility and will make us increase the MAJOR version of Meilisearch SDK.**
+
+**In other terms, if the users MAY have to do more steps than just downloading the new Meilisearch package and running it, a new MAJOR is needed.**
+
+Examples of changes making the code break and then, involving increasing the MAJOR:
+
+- Any change related to the public API the SDK exposes like:
+    - Renaming a method name or classes
+    - Removing or changing a positional argument of a method.
+
+## Reasons why releasing a major version is needed
```

Comment:
I'm not sure I really understand the title
Plus there is one `##` title, and only one `###` title inside. Not sure the `##` title is needed.

Reply:
You're right, I'll remove the H2 title :)